test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -63,6 +63,12 @@ class cursor_bound_01 : public test {
             _inclusive = random_generator::instance().generate_integer(0, 1);
         }
 
+        bound(uint64_t key_size_max, bool lower_bound, char start)
+            : bound(key_size_max, lower_bound)
+        {
+            _key[0] = start;
```

Comment:
This can be part of the initialisation list.

Reply:
Can it? It depends on calling the base constructor.

Reply:
Apparently its not legal C++